assets/js/modules/ads/index.js

```diff
@@ -55,6 +55,7 @@ import { PAX_SETUP_SUCCESS_NOTIFICATION } from './pax/constants';
 import EnhancedConversionsNotification, {
 	ENHANCED_CONVERSIONS_NOTIFICATION_ADS,
 } from './components/notifications/EnhancedConversionsNotification';
+import SettingsDisconnectNote from '@/js/modules/ads/components/settings/SettingsDisconnectNote';
```

Comment:
Let's use a relative path since the imported file is in the same root directory as this one.
```suggestion
import SettingsDisconnectNote from './components/settings/SettingsDisconnectNote';
```